plugins/kubernetes-backend/src/service/KubernetesBuilder.ts

```diff
@@ -273,6 +282,9 @@ export class KubernetesBuilder {
       });
     });
 
+    router.get('/proxy/:encodedQuery', this.makeProxyRequest.bind(this));
+    router.post('/proxy/:encodedQuery', this.makeProxyRequest.bind(this));
```

Comment:
we're only supporting GET and POST here, but the kubernetes API [supports](https://kubernetes.io/docs/reference/using-api/api-concepts/#api-verbs) the full range of HTTP verbs. It's OK to narrow our focus to a few for a first offering, but we should make it explicit (say, in documentation) if we do.